cppForSwig/BtcUtils.h

```diff
@@ -1843,7 +1843,7 @@ class BtcUtils
       CryptoPP::Integer exponent = CryptoPP::Integer::One();
       CryptoPP::Integer five_eight(58);
       CryptoPP::Integer value = CryptoPP::Integer::Zero();
-      for (int i = b58.getSize() - 1; i >= offset; i--)
+      for (size_t i = b58.getSize() - 1; i >= offset; i--)
```

Comment:
This change breaks the code: 

size_t is unsigned, while i is decremented in this loop, and 0 is a valid i, meaning the loop has to exit at -1, but decremeting size_t below 0 lands you at SIZE_MAX.

i has to be a signed value, cast b58.getSize() into a signed type, or copy it into a local signed variable.